content/source/docs/cloud/integrations/service-now/index.html.md

```diff
@@ -9,7 +9,7 @@ description: |-
 
 -> **Integration version:**  v2.0.0
 
--> **Note:** The ServiceNow Catalog integration is designed for use by Terraform Enterprise customers. We do not currently recommend using it with the SaaS version of Terraform Cloud.
+-> **Note:** v1 has been deprecated but can be found [here](service-now-v1/index.html)
```

Comment:
`The v1.x integrations have been deprecated, but documentation for existing installations can be found [here]` maybe? Just a minor tweak

Reply:
Updated